bundles/org.openhab.binding.nuki/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -101,13 +148,38 @@
 				<option value="255">UNDEFINED</option>
 			</options>
 		</state>
+		<command>
+			<options>
+				<option value="1">Unlock</option>
+				<option value="2">Lock</option>
+				<option value="3">Unlatch</option>
+				<option value="4">Lock'n'go</option>
+				<option value="5">Lock'n'go with unlatch</option>
+			</options>
+		</command>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+
+	<channel-type id="smartLockBatteryCharging">
+		<item-type>Switch</item-type>
+		<label>Battery Charging</label>
+		<description>Use this channel to display the current state of charging</description>
+		<category>Energy</category>
+		<state readOnly="true">
+			<options>
+				<option value="OFF">Battery is not charging</option>
+				<option value="ON">Battery is charging</option>
+			</options>
+		</state>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Comment:
What's the reason for setting the auto update policy? This should've only an effect when commands are handled by this channel.